src/day8/re_frame/trace.cljs

```diff
@@ -206,15 +205,14 @@
 
                 (.toFixed duration 1) " ms"]]
               (when show-row?
-                [:tr {:key (str id "-details")}
-                 [:td.trace-details {:col-span 3
+                [:tr {:key (str id "-details")}]))))))
```

Comment:
Was the parens move from line 215 here intentional?

Reply:
oops, a mistake. I fixed it!